contents/docs/integrations/python-integration.md

```diff
@@ -212,13 +212,14 @@ POSTHOG_DJANGO = {
 
 ## Naming Confusion
 
-As our open source project [PostHog](https://github.com/PostHog/posthog) shares the same module name, we created a special `posthog-analytics` package, mostly for internal use to avoid module collision. It is the exact same.
+As our open source project [PostHog](https://github.com/PostHog/posthog) shares the same module name, we created a special `posthoganalytics` package, mostly for internal use to avoid module collision. It is the exact same.
 
 ## How to Release
```

Comment:
Not caused by this PR but we should move this to the Handbook - not good to have internal instructions mixed with Docs